introduction.md

```diff
@@ -0,0 +1 @@
+My name is Liang Zhang, github account liangnuaa.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I am going to try the front end things.